### PR TITLE
Python REST API example, document Scripts plugin, Scripts plugin example

### PR DESCRIPTION
**Pull request by fdlk**

### CODE REVIEW

File: molgenis_documentation/REST API.md [hunk 1]
```diff
@@ -146,56 +157,231 @@ By default, the MOLGENIS R API retrieves its data from the server you retrieved
...
+pip install matplotlib
+```
+
+Start an interactive python shell and create a molgenis connection:
 
+```python
+import molgenis
+c = molgenis.Connection("https://molgenis56.target.rug.nl/api/")
+```
+This loads the R API and points it at the molgenis56 server. If you take a look at the connection by typing
```
Comment (Mark-de-Haan) [inline molgenis_documentation/REST API.md]: Its the Python api right? :p

File: molgenis_documentation/REST API.md [hunk 2]
```diff
@@ -146,56 +157,231 @@ By default, the MOLGENIS R API retrieves its data from the server you retrieved
...
 
+Internally, MOLGENIS uses `molgenis.RestClient` and `molgenis.RestClientV2` defined in `molgenis.js`. Their methods are implemented using `jquery.ajax`. This interface is not yet supported externally, but if you're planning to access MOLGENIS from JavaScript, you'll want to take a look at that for inspiration.
+
 ## HTTP
 
+This is the raw HTTP REST API that the other APIs are built upon.
+There are two versions, v1 and v2. v2 should be seen as an extension of v1.
+
+The main differences between v2 and v1 are that v2 retrieves more entity metadata and that it allows you to query using RSQL.
+But beware: the v1 csv endpoint already queries using RSQL.
```
Comment (Mark-de-Haan) [inline molgenis_documentation/REST API.md]: Meaning?

File: molgenis_documentation/REST API.md [hunk 3]
```diff
@@ -146,56 +157,231 @@ By default, the MOLGENIS R API retrieves its data from the server you retrieved
...
-The Scripting plugin allows you to edit the available scripts.
-You can also execute them by pushing the green arrows button.
+The Scripts plugin allows you to create, edit and delete the available scripts.
+
+## Creating a script
 
-The Scripting plugin also shows you a list of parameters.
+You open the script editor by clicking the add button ![image](images/new.png).
 
+### Name
```
Comment (Mark-de-Haan) [inline molgenis_documentation/REST API.md]: I wouldn't use header (#) notation for form fields. Maybe use asterisks (*) to create a list?

Comment (fdlk) [inline molgenis_documentation/REST API.md]: I didn't know much more to write about the name attribute, but the other attributes then turned out to be worth their own paragraphs. So leaving Name with a * and then moving on to ### for the rest seemed a bit weird.

File: molgenis_documentation/REST API.md (hunk 1 above)
Comment (fdlk) [inline molgenis_documentation/REST API.md]: Whoops! :D

File: molgenis_documentation/REST API.md (hunk 2 above)
Comment (fdlk) [inline molgenis_documentation/REST API.md]: Big todo here

**Pull request merged**